src/components/Layout/Layout.component.tsx

```diff
@@ -256,7 +256,10 @@ export const Layout = ({ maxWidth = true }: { maxWidth?: boolean }) => {
 	const disableMaxWidth = isChat;
 
 	return (
-		<Container maxWidth={disableMaxWidth ? false : maxWidth ? 'desktop' : false} className="main-container">
+		<Container
+			maxWidth={disableMaxWidth ? false : maxWidth ? 'desktop' : false}
+			className="main-container"
+		>
```

Comment:
- boolean naming
- disableMaxWidth seems like redundant

```suggestion
		<Container
			maxWidth={isMaxWidth ? 'desktop' : false}
			className="main-container"
		>
```